README.md

```diff
@@ -51,15 +51,19 @@ children = [
   {Finch,
    name: MyConfiguredFinch,
    pools: %{
-     :default => [size: 10],
+     :default => [size: 10, count: 2],
      "https://hex.pm" => [size: 32, count: 8]
    }}
 ]
 ```
 
 Pools will be started for each configured `{scheme, host, port}` when Finch is started.
 For any unconfigured `{scheme, host, port}`, the pool will be started the first time
-it is requested. Note pools are not automatically terminated by default, if you need to
+it is requested using the `:default` configuration. This means given the pool
+configuration above each origin/`{scheme, host, port}` will launch 2 (`:count`) new pool
```

Comment:
This is more general and I can remove it again, but I think people may be familiar with the term of [origin](https://en.wikipedia.org/wiki/Same-origin_policy#Origin_determination_rules) and it may be useful in both the docs and maybe code base (aka calling the 3-tuple scheme, host, port origin).